README.md

```diff
@@ -1,6 +1,11 @@
 # DSpace-Docker-Images
 
-This repository contains the source code for Docker Images for the [DSpace](https://dspace.org) project.
+This repository contains docker resources for the [DSpace](https://dspace.org) project.
+- Docker Compose files for DSpace 4x and 5x
```

Comment:
Minor addition/improvement. I'd suggest we note here that the Docker Compose files for 6.x and 7.x are in the main DSpace codebase. (Just in case that is not self evident)